7024Movies/Pages/DiscoverOther.cshtml

```diff
@@ -0,0 +1,33 @@
+﻿@page
+@model _7024Movies.Pages.DiscoverOtherModel
+@{
+    var others = (DiscoverM.Result[])ViewData["Others"];
+    List<MovieId.MovieIds> otherids = (List<MovieId.MovieIds>)ViewData["OtherLkp"];
+    List<IMDB.MovieImdb> otherimdb = (List<IMDB.MovieImdb>)ViewData["OtherIMDB"];
+
+}
+
+<head>
+    <link rel="stylesheet" href="https://drvic10k.github.io/bootstrap-sortable/Contents/bootstrap-sortable.css" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.js"></script>
+    <script src="https://drvic10k.github.io/bootstrap-sortable/Scripts/bootstrap-sortable.js"></script>
+</head>
+
+<div class="row padbottom">
+    <h1 class="center">Top Other Movies</h1>
+</div>
```

Comment:
I added a new Razor Page called DiscoverOther that contains the same display features and follows the same C# as the other two Razor Pages that allows for integration of the Json for these movies if they decide to incorporate